lib/helpers/rgba.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  getRgba: getRgba
+};
+
+function getRgba(imageStep) {
+  var rgba = {
```

Comment:
If `background` is undefined, return `null` here. This will allow the defaults of each different operation to kick in.